tests/write_font_test.py

```diff
@@ -41,6 +41,34 @@ def test_keep_glyph_names(svgs, color_format, keep_glyph_names):
         assert ttfont["post"].formatType == 3.0
         assert ufo.glyphOrder != ttfont.getGlyphOrder()
 
+@pytest.mark.parametrize(
+    "svgs", [("rect.svg", "one-o-clock.svg")]
+)
+@pytest.mark.parametrize(
+    "color_format", ["glyf", "cff2_colr_0", "cff2_colr_1", "cff_colr_0", "cff_colr_1", "glyf_colr_0", "glyf_colr_1", "picosvg", "untouchedsvg"]
```

Comment:
Do we need to test all font formats?

Reply:
i don't think so, the version setting is orthogonal to the color font format and applies to all of them indistinctly as they all will have a head table and a name table, like any opentype fonts. I think you can just test a couple, like glyf_colr_1 and cff_colr_1

Reply:
Done, PTAL.